.env

```diff
@@ -1,4 +1,12 @@
 DB_NAME=campus_db
 DB_ROOT_PASSWORD=secret_root_password
 DB_PORT=3306
-ENVIRONMENT=dev
\ No newline at end of file
+
+ENVIRONMENT=dev
+
+INFLUXDB_USER=campus_backend
+INFLUXDB_PASSWORD=secret_influxdb_password
+INFLUXDB_ORG=tum
+INFLUXDB_BUCKET=campus_backend
+INFLUXDB_URL=http://influxdb:8086
+INFLUXDB_ADMIN_TOKEN=secret_influxdb_admin_token
```

Comment:
Bight be worth documenting those here?
https://github.com/TUM-Dev/Campus-Backend#environment-variables